ykcapi/yk_testing.h

```diff
@@ -23,16 +23,16 @@ void __yktrace_drop_irtrace(void *trace);
 
 // Blocks the compiler from optimising the specified value or expression.
 //
-// This is the non-const variant borrowed from Google benchmark:
+// This is similar to the non-const variant borrowed from Google benchmark:
 // https://github.com/google/benchmark/blob/e451e50e9b8af453f076dec10bd6890847f1624e/include/benchmark/benchmark.h#L350
 //
-// Note that Google benchmark also defines a variant of this for constant data.
-// At the time of writing, the below NOOPT seems to suffice (even for
-// constants), but we may need to borrow the const version later.
-static inline void NOOPT(void *x) {
+// Our version works on a value, rather than a pointer.
+//
+// Note that Google benchmark also defines a variant for constant data. At the
```

Comment:
s/note that Google/note that the Google/

Reply:
For better or worse, "Google benchmark" is the name of the project.

https://github.com/google/benchmark

Reply:
Then it should presumably be "Google Benchmark" (capital 'b')?

Reply:
72060fb